phpstan-baseline.neon

```diff
@@ -0,0 +1,22 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^The Composer\\\\Semver\\\\Constraint\\\\AbstractConstraint abstract class is deprecated, there is no replacement for it, it will be removed in the next major version\\.$#"
+			count: 1
+			path: src/Constraint/AbstractConstraint.php
+
+		-
+			message: "#^Call to an undefined method Composer\\\\Semver\\\\Constraint\\\\AbstractConstraint\\:\\:matchSpecific\\(\\)\\.$#"
+			count: 1
+			path: src/Constraint/AbstractConstraint.php
```

Comment:
Those probably can be removed if you rebase, I deleted this deprecated class from master now.

Reply:
rebased and regenerated baseline.

I created a mess with commits in this PR.
at best you just 'suqash and merge' it, to get a clean history